pages/edge.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Page from '../components/edge/Page';
+import Download from '../components/edge/Download';
+import Link from '../components/edge/Link';
+import Gain from '../components/edge/Gain';
+import { withTranslation } from '../i18n';
+import { string } from 'prop-types';
```

Comment:
Could you remove this? Looks like 'string' is declared but its value is never read.